channels/api/grpc/request.go

```diff
@@ -3,7 +3,13 @@
 
 package grpc
 
-import "github.com/absmach/supermq/pkg/connections"
+import (
+	"github.com/absmach/supermq/pkg/connections"
+	"github.com/absmach/supermq/pkg/errors"
+	"github.com/absmach/supermq/pkg/policies"
+)
+
+var errDomainID = errors.New("domain id required for user type")
```

Comment:
Let's make messages less cryptic: `domain ID required for users`.

Reply:
Changed